### PR TITLE
Addition of guide to how to download dependencies

### PR DESCRIPTION
**Pull request by Kerveyfelix**
To make this Repository more user friendly, I have added resources for installing dependencies.
I have made the following changes in the README.md file:
1) Added the link to download the Python 3.x version for Python's site.
2)  Add the commands to add Requests and Flask package in Python.

### CODE REVIEW

**Review by martyndavies — CHANGES_REQUESTED** [review]
Thanks for your contribution. You have spelling mistakes on the word 'install' in the added text, please can you fix that and recommit the changes.

**Comment by Kerveyfelix** [comment]
I have rectified the spelling mistakes. Kindly review @martyndavies

File: README.md [hunk 1]
```diff
@@ -3,9 +3,11 @@ Dial a Christmas Carol - for Advent Calendar
 
 You need to have the following installed:
 
-* Python 3.x
-* Request
+[Python 3.x](https://www.python.org/downloads/)
+* Request 
+-Use the following command in your terminal to install Request: pip install requests
```
Comment (martyndavies) [inline README.md:8]: We use backticks to highlight commands that people should execute, for example `pip install requests`, please can you update to include these?

**Review by martyndavies — CHANGES_REQUESTED** [review]

**Comment by martyndavies** [comment]
@Kerveyfelix Are you going to be able to make these changes?

**Comment by Kerveyfelix** [comment]
@martyndavies I am making the changes now.

**Comment by Kerveyfelix** [comment]
@martyndavies  Can you review my PR? I have added backticks.

**Review by martyndavies — APPROVED** [review]

**Pull request merged**